### PR TITLE
final commit

### PR DESCRIPTION
**Pull request by nehalskdev**

### CODE REVIEW

**Review by ManSOorcode — APPROVED** [review]
LGTM keep going 👍

**Review by murtazshaikh — APPROVED** [review]
LGTM!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Border Animation</title>
+</head>
+<body>
+    <div class="frem">
```
Comment (Abbas4tech) [inline index.html:11]: Please use some meaningful name for class

**Review by Abbas4tech — APPROVED** [review]
Fix mention thing, else LGTM

File: index.html (hunk 1 above)
Comment (nehalskdev) [reply index.html:11]: surely I have noted your suggestion I will implement it from now on.

**Pull request merged**